zcash_client_backend/src/data_api.rs

```diff
@@ -201,6 +202,7 @@ pub struct ReceivedTransaction<'a> {
 /// The purpose of this struct is to permit atomic updates of the
 /// wallet database when transactions are created and submitted
 /// to the network.
+#[derive(Clone)]
```

Comment:
Woudln't it be problem when we clone this and use `store_sent_tx` or `store_received_tx` ? Can't `Arc<Mutex>` be better?

Reply:
yeah..will update, thanks